libindi/drivers.xml

```diff
@@ -41,6 +41,10 @@
                 <driver name="LX200 GPS">indi_lx200gps</driver>
                 <version>2.0</version>
         </device>
+        <device label="EQ500X" manufacturer="Meade">
```

Comment:
Made by Meade?

can this be made to indi_eq500x_telescope as well to be consistent with the newer drivers?

Reply:
Thanks, fixed.